pmd-core/src/main/java/net/sourceforge/pmd/lang/document/RootTextDocument.java

```diff
@@ -69,24 +69,27 @@ public Chars getText() {
 
     @Override
     public FileLocation toLocation(TextRegion region) {
-        checkInRange(region, this.getLength());
-        SourceCodePositioner positioner = content.getPositioner();
+    checkInRange(region, this.getLength());
+    SourceCodePositioner positioner = content.getPositioner();
+
+    // We use longs to return both numbers at the same time
+    // This limits us to 2 billion lines or columns, which is FINE
+    TextPos2d bpos = positioner.lineColFromOffset(region.getStartOffset(), true);
+    TextPos2d epos = region.isEmpty() ? bpos
+                                      : positioner.lineColFromOffset(region.getEndOffset(), false);
+
+    FileLocation location = new FileLocation(
```

Comment:
Apart from the whitespace changes you just added an unnecessary local variable here.